packages/lucide-react/build.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { nodeFs } from '@file-services/node';
+import * as lucideReact from 'lucide-react';
+
+interface IconData {
+    name: string;
+    aliases: string[];
+    svgString: string;
+}
+
+interface IconListEntry {
+    name: string;
+    aliases: string[];
+    file: string;
+}
+
+interface IconTypeAugmentation {
+    name: string;
+    aliases: string[];
+    file: string;
+}
+
+interface TypeAugmentationFile {
+    icons: IconTypeAugmentation[];
+}
+
+function buildIcons(): void {
+    const distDir = nodeFs.join(__dirname, 'dist');
+    const svgDir = nodeFs.join(distDir, 'svg');
+
+    nodeFs.ensureDirectorySync(distDir);
+    nodeFs.ensureDirectorySync(svgDir);
+
+    const svgsByContent = new Map<string, IconData>();
+
+    for (const [exportName, IconComponent] of Object.entries(lucideReact)) {
```

Comment:
lucide-react exports the same icon 3 times:
- Speaker
- LucideSpeaker
- SpeakerIcon

So while we could list all 3, we could just list "Speaker", and use the "SpeakerIcon" export (for less code confusion)